### PR TITLE
CDK: add native gas token address to contract deployment step

### PR DESCRIPTION
**Pull request by kmurphypolygon**
This PR adds a how to explaining how to add a native gas token to your CDK deployment.

https://docs.google.com/document/d/1fWivzbe0CxX1wrb3mXdJjqJ-DKQiont4uTm-0RUCcHY/edit

### CODE REVIEW

File: docs/cdk/get-started/deploy-rollup/prerequisites.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 !!! info "Computing resources"
     - Running a full prover is an enterprise level installation and extremely resource-heavy. 
     - These instructions run the mock prover instead.
+    - Check out the [configure prover](configure-prover.md#configure-the-prover) section for more information.
```
Comment (kmurphypolygon) [inline docs/cdk/get-started/deploy-rollup/prerequisites.md]: this is a clarifying statement coming from another query...

File: docs/cdk/how-to/use-native-token.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "":"",
+    "maxPriorityFeePerGas":"",
+    "multiplierGas": ""
+    "gasTokenAddress": "<TOKEN-ADDRESS>"
+}
+```
+
+`TOKEN-ADDRESS` is the address of a deployed ERC20 L1 token.
+
+If you leave the value empty, the ETH token address is used as default.
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/use-native-token.md]:
```suggestion
If you leave the value empty, the ETH token address is used as a default.
```

File: docs/cdk/version-matrix.md [hunk 3]
```diff
@@ -55,6 +55,16 @@ The table below shows the version compatibility for CDK releases and related com
...
+    <td>Etrog + uLxLy</td>
+    <td>?</td>
+    <td><a href="https://github.com/0xPolygonHermez/zkevm-node">zkevm-node@v0.5.0-RCXX (where XX=latest tag number)</a></td>
+    <td><a href="https://github.com/0xPolygon/cdk-validium-node">cdk-validium-node@v0.5.0-RC4 </a></td>
+    <td><a href="https://github.com/0xPolygon/cdk-data-availability">cdk-data-availability@ feature/etrog</a></td>
+    <td><a href="https://github.com/0xPolygonHermez/zkevm-prover/releases/tag/v4.0.4">zkevm-prover@v4.0.4</a></td>
+    <td><a href="https://github.com/0xPolygonHermez/zkevm-contracts/tree/feature/build-v4.0.0-rc.5-fork.7">zkevm-contracts@v4.0.0-rc.5-fork.7</a></td>
+    <td><a href="https://github.com/0xPolygonHermez/zkevm-bridge-service/tree/v0.4.0-RC4">zkevm-bridge-service@v0.4.0-rc4</a></td>
+  </tr>
 </tbody>
```
Comment (kmurphypolygon) [inline docs/cdk/version-matrix.md]: this extra row may not be correct but i thought i'd add it to see

File: docs/cdk/how-to/use-native-token.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+    !!! warning
+        The directory naming is dependent on your build type (rollup or validium). 
+
+2. Open the `deploy_parameters.json` file and add a new entry:
+
+    ```json
+    {
+        "":"",
+        "":"",
```
Comment (hsutaiyu) [inline docs/cdk/how-to/use-native-token.md:26]: Empty fields here intentional?

File: docs/cdk/how-to/integrate-agglayer.md [hunk 5]
```diff
@@ -75,4 +76,7 @@ Add the chain configurations to the `../agglayer/docker/data/agglayer.toml` file
 
 * Configure `[FullNodeRPCs]` to point to the corresponding L2 full node.
 * Configure `[L1]` to point to the corresponding L1 chain.
-* Configure the `[DB]` section with the managed database details.
\ No newline at end of file
+* Configure the `[DB]` section with the managed database details.
```
Comment (hsutaiyu) [inline docs/cdk/how-to/integrate-agglayer.md]: Maybe include an example configuration with all valid keys for these tables in the .toml file for clarity?

**Review by hsutaiyu — APPROVED** [review]
Approved with suggestions.

File: docs/cdk/how-to/use-native-token.md (hunk 4 above)
Comment (kmurphypolygon) [reply docs/cdk/how-to/use-native-token.md:26]: yes, added three dots to clarify

File: docs/cdk/how-to/integrate-agglayer.md (hunk 5 above)
Comment (kmurphypolygon) [reply docs/cdk/how-to/integrate-agglayer.md]: that would be ideal

Comment (kmurphypolygon) [reply docs/cdk/how-to/integrate-agglayer.md]: i'm going to leave it for now because it's fairly self-explanatory

**Pull request merged**